Integrations/GoogleCloudTranslate/README.md

```diff
@@ -0,0 +1,7 @@
+## GoogleCloudTranslate
```

Comment:
this file should be generated using `HTMLDocsAutomation` script: https://demisto.pan.dev/docs/integration-docs

Reply:
Improved in the new commit